### PR TITLE
Write to summary field source.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review

### CODE REVIEW

**Review by geirst — DISMISSED** [review]
👍

File: config-model/src/main/java/com/yahoo/schema/processing/TypedTransformProvider.java [hunk 1]
```diff
@@ -39,9 +39,17 @@ else if (exp instanceof IndexExpression) {
             }
             else if (exp instanceof SummaryExpression) {
                 Field field = schema.getSummaryField(fieldName);
-                if (field == null)
-                    throw new IllegalArgumentException("Summary field '" + fieldName + "' not found.");
-                fieldType = field.getDataType();
+                if (field == null) {
+                    // Use document field is summary field is not found
```
[the remark this answers is not in the document]
Comment (toregge) [reply config-model/src/main/java/com/yahoo/schema/processing/TypedTransformProvider.java]: Changed.

File: config-model/src/main/java/com/yahoo/schema/processing/IndexingOutputs.java [hunk 2]
```diff
@@ -137,6 +137,10 @@ protected Expression doConvert(Expression exp) {
                 for (String fieldName : summaryFields) {
                     ret.add(new SummaryExpression(fieldName));
                 }
+                // Write to summary field source
```
[the remark this answers is not in the document]
Comment (toregge) [reply config-model/src/main/java/com/yahoo/schema/processing/IndexingOutputs.java]: Done.

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**